src/main/java/ua/java/caesar/caesarcypher/BruteForce.java

```diff
@@ -0,0 +1,108 @@
+package ua.java.caesar.caesarcypher;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BruteForce {
+    private final Map<Character, Integer> MOST_USED_LETTERS = new HashMap<>();
+
+    public BruteForce() {
+        MOST_USED_LETTERS.put('e', 0);
+        MOST_USED_LETTERS.put('t', 0);
+        MOST_USED_LETTERS.put('a', 0);
+    }
+
+
+    public void writePossibleDecryptionsWithKeys(CaesarCypher caesarCypher, String sourcePath) {
+        String newFileName = getNewFileName(sourcePath);
+        byte[] bytes = readAllBytes(sourcePath);
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        decryptWithPassingKeys(caesarCypher, bytes, outputStream);
+        writeDecryprions(newFileName, outputStream);
+    }
+
+    private void writeDecryprions(String newFileName, ByteArrayOutputStream outputStream) {
+        try {
+            Files.write(Paths.get(newFileName), outputStream.toByteArray());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private byte[] readAllBytes(String sourcePath) {
+        byte[] bytes;
+        try {
+            bytes = Files.readAllBytes(Paths.get(sourcePath));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Maybe better here rethrow exception using approach as you have already implemented in writeDecryprions method